File: problem1/python/euler1.py
#!/usr/local/bin/python3


def sum_multiples_3_5(max=10):
  sum = 0
  for x in range (1,max):
    if x % 3 == 0 or x % 5 == 0:
      sum+=x
  return sum


if __name__ == "__main__":
  print( sum_multiples_3_5(1000))
   
    

File: problem1/go/euler1.go
package main

import (
   "fmt"
)

func Multiples_3_5(max int) int {

  sum := 0
  for i := 1; i < max; i++{
    if i%3 == 0 || i%5 ==0 {
      sum+=i
    }
  } 
  return sum
}

func main() {
  fmt.Println(Multiples_3_5(1000))

}
